src/stream-tcp.c

```diff
@@ -5849,6 +5849,21 @@ static inline int StreamTcpValidateAck(TcpSession *ssn, TcpStream *stream, Packe
     SCReturnInt(-1);
 }
 
+/** \brief update reassembly progress
+
+ * \param ssn TCP Session
+ * \param direction direction to set the flag in: 0 toserver, 1 toclient
+ */
+void StreamTcpUpdateAppLayerProgress (TcpSession *ssn, char direction,
+        const uint32_t progress)
```

Comment:
Style `StreamTcpUpdateAppLayerProgress (`

Reply:
Missing the issue.

Reply:
Whitespace between function name and arguments.